packages/cubejs-api-gateway/src/gateway.ts

```diff
@@ -1236,18 +1236,18 @@ class ApiGateway {
   }
 
   private parseMemberExpression(memberExpression: string): string | MemberExpression {
-    const match = memberExpression.match(memberExpressionRegex);
-    if (match) {
-      const args = match[3].split(',');
-      args.push(`return \`${match[4]}\``);
+    try {
```

Comment:
I still feel that this large `try` block can catch something unrelated to `JSON.parse`, but this may be irrelevant if we consider that any failure must return `memberExpression` as is.